packages/langium-vscode/esbuild.js

```diff
@@ -20,7 +20,7 @@ require('esbuild').build({
     loader: { '.ts': 'ts' },
     external: ['vscode'], // the vscode-module is created on-the-fly and must be excluded.
     platform: 'node', // VSCode extensions run in a node process
-    sourcemap: true,
+    sourcemap: !minify,
```

Comment:
Should we use different settings for dev and production?

Reply:
Since `dev` is effectively without minification, I don't really see a reason right now.